_posts/2022-07-25-Building-CNF-Using-CNDP-Rust.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: Building Cloud Native Network Functions using Rust
+author: CNDP Team
+excerpt_separator: <!--more-->
+---
+
+
+## Rust for Networking and Cloud Native Development
+
+The [Rust](#rust) programming language has been steadily gaining
+popularity over the last several years. Many developers, companies use
+Rust in production today for fast, low-resource, cross-platform
+solutions. Rust is fast and memory efficient, with no runtime or garbage
+collector. It also has a rich type-system and ownership model with
+guarantees for memory and thread safety which eliminates many classes of
+bugs at compile-time. Rust's predictable performance, low resource
+footprint, reliability, security, and fearless concurrency at scale
+makes it a good candidate for building Cloud native Network Functions
+(CNFs).
+
+There is a growing list of cloud native applications and services built
+using Rust. [AWS firecracker](#aws-fire-cracker) and [Linkerd service mesh

```

Comment:
The link behavior is kind of odd. When I click, my browser scrolls down as far as it can, which is the end of the page. I expect the thing which was linked to appear "in focus" or somewhere near the top of the screen. This isn't the case, because the screen is already scrolled down as far as it can go. Without duplicating the link, I'm not sure how you can fix htis behavior.

I guess its fine to leave it the way it is. Once the reader realizes you're linking to the "references" sectino, they will stop clicking the links.

Reply:
Yeah. I too noticed it. Tried couple of things. But when we click the link it was always scrolls to reference section.